### PR TITLE
CDAP-15816: fix collision of release branch ahead develop

### PR DESCRIPTION
**Pull request by AntonGensitskiy**
Release branch is ahead develop branch, this breaks develop flow.
In scope of this PR:

- fix collision of release branch ahead develop
- updated version and package name

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -58,24 +58,25 @@
...
-    <cdap.version>4.1.0</cdap.version>
+    <cdap.version>6.0.0-SNAPSHOT</cdap.version>
+    <guava.version>19.0</guava.version>
     <hadoop.version>2.3.0</hadoop.version>
-    <hydrator.version>1.7.0</hydrator.version>
+    <hydrator.version>2.2.0-SNAPSHOT</hydrator.version>
     <widgets.dir>widgets</widgets.dir>
     <docs.dir>docs</docs.dir>
-    <app.parents>system:cdap-etl-batch[4.1.0-SNAPSHOT,5.0.0-SNAPSHOT),system:cdap-etl-realtime[4.1.0-SNAPSHOT,5.0.0-SNAPSHOT),system:cdap-data-pipeline[4.1.0-SNAPSHOT,5.0.0-SNAPSHOT),system:cdap-data-streams[4.1.0-SNAPSHOT,5.0.0-SNAPSHOT)</app.parents>
+    <app.parents>system:cdap-data-pipeline[6.0.0-SNAPSHOT,7.0.0-SNAPSHOT),system:cdap-data-streams[6.0.0-SNAPSHOT,7.0.0-SNAPSHOT)</app.parents>
```
Comment (aonischuk) [inline pom.xml]: This is not used I think. The same goes for widgets.dir and docs.dir.
I think these are some legacy properties, which care not needed now.

File: pom.xml [hunk 2]
```diff
@@ -58,24 +58,25 @@
...
     <!-- this is here because project.basedir evaluates to null in the script build step -->
     <main.basedir>${project.basedir}</main.basedir>
```
Comment (aonischuk) [inline pom.xml]: not used

File: pom.xml [hunk 3]
```diff
@@ -58,24 +58,25 @@
...
   <properties>
     <jrecord.version>0.80.8</jrecord.version>
     <cb2xml.version>0.80.8-jrecord</cb2xml.version>
     <main.basedir>${project.basedir}</main.basedir>
-    <cdap.version>4.1.0</cdap.version>
+    <cdap.version>6.0.0-SNAPSHOT</cdap.version>
+    <guava.version>19.0</guava.version>
     <hadoop.version>2.3.0</hadoop.version>
-    <hydrator.version>1.7.0</hydrator.version>
+    <hydrator.version>2.2.0-SNAPSHOT</hydrator.version>
```
Comment (aonischuk) [inline pom.xml]: Can be 2.2.0 no need for SNAPSHOT

File: pom.xml [hunk 4]
```diff
@@ -58,24 +58,25 @@
...
   <issueManagement>
-    <url>https://issues.cask.co/browse/HYDRATOR</url>
+    <url>https://issues.cask.co</url>
   </issueManagement>
   <properties>
     <jrecord.version>0.80.8</jrecord.version>
     <cb2xml.version>0.80.8-jrecord</cb2xml.version>
     <main.basedir>${project.basedir}</main.basedir>
-    <cdap.version>4.1.0</cdap.version>
+    <cdap.version>6.0.0-SNAPSHOT</cdap.version>
```
Comment (aonischuk) [inline pom.xml]: If we plan to use new validations here. We need 6.1.0-SNAPSHOT

File: README.md [hunk 5]
```diff
@@ -26,7 +29,7 @@ Getting Started
 
 Prerequisites
 -------------
-CDAP version 3.5.x or higher. 
+CDAP version 6.0.x or higher.
```
Comment (aonischuk) [inline README.md]: 6.1.x

**Comment by aonischuk** [comment]
Also this plugin lacks cdap-maven-plugin usage.

File: pom.xml (hunk 1 above)
Comment (AntonGensitskiy) [reply pom.xml]: this property used maven-antrun-plugin in javascript

File: pom.xml (hunk 2 above)
Comment (AntonGensitskiy) [reply pom.xml]: same as previous

File: pom.xml (hunk 3 above)
Comment (AntonGensitskiy) [reply pom.xml]: I will change the hydrator.version

File: README.md (hunk 5 above)
Comment (AntonGensitskiy) [reply README.md]: same as previous

**Comment by AntonGensitskiy** [comment]
> Also this plugin lacks cdap-maven-plugin usage.

In this case maven-antrun-plugin should be removed, correct?

**Comment by aonischuk** [comment]
The javascript code for creating the plugin json should be removed. As this approach is obsolete. Use cdap-maven-plugin instead. And follow the comments regarding removing the variables posted above.

File: pom.xml (hunk 1 above)
Comment (aonischuk) [reply pom.xml]: we need to remove javascript usage. Please read the comment below.

File: pom.xml (hunk 2 above)
Comment (aonischuk) [reply pom.xml]: the same

**Comment by AntonGensitskiy** [comment]
All change was done.

**Pull request merged**